src/kv/test/kv_test.cpp

```diff
@@ -2464,6 +2512,8 @@ TEST_CASE("Store clear")
 TEST_CASE("Reported TxID after commit")
 {
   kv::Store kv_store;
+  auto encryptor = std::make_shared<kv::NullTxEncryptor>();
+  kv_store.set_encryptor(encryptor);
```

Comment:
Not needed since you've added these lines. But next time we make something mandatory on the `Store`, or the boiler plate changes, we should probably just replace all of these with a `kv::test::Store` whose constructor sets null encryptors, stub consensuses, etc etc.

Reply:
I did contemplate doing that in Store, as mentioned above, but at least for this particular task it wouldn't have paid off: setting a default null encryptor produces painfully surprising digests rather than an early exception. Having a `make_test_store()` that does this explicitly when wanted is definitely a good step, I agree.